doc/source/whatsnew/v1.6.0.rst

```diff
@@ -212,6 +212,7 @@ Sparse
 ExtensionArray
 ^^^^^^^^^^^^^^
 - Bug in :meth:`Series.mean` overflowing unnecessarily with nullable integers (:issue:`48378`)
+- Loss of dtype information when concatenating EA with empty EA of same type (:issue:`48510`)
```

Comment:
`EA` -> `ExtensionArray`

Also could you make "Loss of dtype" more specific to describe that the result dtype is object?